extensions/amp-social-share/1.0/social-share.js

```diff
@@ -52,23 +52,31 @@ export function SocialShare(props) {
 
   const type = props['type'].toUpperCase();
   const baseStyle = CSS.BASE_STYLE;
-  const backgroundStyle = CSS[type] || CSS.DEFAULT;
+  const backgroundStyle = CSS[type];
   const size = {
     width: checkedWidth,
     height: checkedHeight,
   };
+
+  // Only show children if not using a pre-configured type
+  const children = getSocialConfig(props['type']) ? (
```

Comment:
What do you think about passing `props['children']` to `SocialShareIcon` and having the default case return `props['children']` instead of the current `<svg {...props}></svg>`? That way we don't need this ternary here.

Reply:
I like the idea.  Although while trying the implementation, I keep finding that the children passed in from the publisher are not passed into `props['children']` and if passed into the `SocialShareIcon` they are not rendered there.  Instead they are rendered as a sibling to the highest level `<div>` returned by the component (which looks fine from a rendering perspective).  Maybe @jridgewell can comment if there's a better way to do this.